src/components/ControlledAnimated.tsx

```diff
@@ -12,13 +12,33 @@ const controlledAnimated = <A extends string, T extends HTMLTags = 'div'>(
         animations,
         currentAnimation,
         onAnimationEnd,
+        finishOnInterrupt = false,
+        cancelOnInterrupt = false,
```

Comment:
Discovered in testing that cancel() happens by default, so going to remove this prop